src/db/add_package.rs

```diff
@@ -244,6 +244,17 @@ pub(crate) async fn finish_build(
     debug!("updating build after finishing");
     let hostname = hostname::get()?;
 
+    let rustc_date = match parse_rustc_date(rustc_version) {
```

Comment:
Do we not want to return early in case the date couldn't be parsed instead?

Reply:
I wasn't sure about the possible edge cases. While I tested the parsing with all existing `rustc_version` contents in our production `builds` table, we have a handful of records where the version is empty. 

So for now I wanted to just log an error. In the end, the build finished, even when the version couldn't be parsed.

Reply:
Oh well. It's fine I suppose then. Please add a comment with this explanation in the code then so future us won't wonder what's going on. ;)

Reply:
added a comment